scripts/gwas_to_genes.py

```diff
@@ -39,8 +39,8 @@
 import json
 import cPickle as pickle
 import time
-import pdb
 import subprocess
+import ipdb
```

Comment:
Not especially - though ipdb is better. I've added every file in my directory to commit which makes this quite messy... noted.